src/sentry/api/endpoints/organization_auth_providers.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import absolute_import
+
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.auth import manager
+from sentry.auth.providers.saml2 import SAML2Provider, HAS_SAML2
+from sentry.api.bases.organization import OrganizationEndpoint, OrganizationAdminPermission
+from sentry.api.serializers import serialize
+
+
+class OrganizationAuthProvidersEndpoint(OrganizationEndpoint):
+    permission_classes = (OrganizationAdminPermission, )
+
+    def get(self, request, organization):
+        """
+        List an Organization's auth providers
```

Comment:
is this listing what they could set up or what they have set up? i'd maybe just clarify here if it's the former